components/CompanyLogos.js

```diff
@@ -1029,11 +1029,347 @@ export const WazirXLogo = () => (
-      <path d="m371.81 163.76c-0.67188 1.7812-3.6172 2.6875-5.8398 2.6875-2.1758 0-3.625-0.86328-4.3594-2.6016-0.73828-1.7266-1.0977-5.6719-1.0977-11.848v-37.137c0-6.3984 0.32031-10.391 0.96875-11.984 0.64062-1.5742 2.0469-2.375 4.2227-2.375 2.2266 0 5.207 0.90234 5.9766 2.7188 0.76172 1.8164 1.1445 5.6992 1.1445 11.641v36.008c-0.23047 7.4023-0.56641 11.695-1.0156 12.891zm-46.465 21.207h32.863c1.3672-5.3672 2.1211-8.3516 2.2734-8.9531 2.9766 3.6016 6.2461 6.3047 9.8398 8.0977 3.5742 1.7969 8.9258 2.6953 13.07 2.6953 5.7695 0 10.746-1.5117 14.945-4.543 4.1914-3.0273 6.8633-6.6094 8-10.73 1.1367-4.1289 1.7031-10.398 1.7031-18.832v-39.422c0-8.4805-0.19141-14.016-0.56641-16.617-0.375-2.6016-1.4961-5.25-3.3594-7.9609-1.8672-2.7109-4.5781-4.8164-8.1289-6.3203-3.5508-1.5039-7.7461-2.2539-12.578-2.2539-4.1992 0-9.5742 0.83984-13.16 2.4961-3.5742 1.6562-6.8203 4.168-9.7344 7.5352v-45.711h-35.168z"/>
+      <path d="m160.96 111.32c-3.1367-21.418-4.8828-33.32-5.2266-35.695-1.5273-11.465-2.9805-21.441-4.375-29.953h-47.328v140.52h31.977l0.11328-92.785 13.453 92.785h22.777l12.762-94.848 0.11719 94.848h31.875v-140.52h-47.691z" />
+      <path d="m277.37 74.902c0.40234 1.793 0.60937 5.8594 0.60937 12.211v54.48c0 9.3516-0.60937 15.078-1.8164 17.191-1.2148 2.1094-4.4492 3.1602-9.6875 3.1602v-92.242c3.9766 0 6.6875 0.42578 8.1289 1.2578 1.4375 0.83984 2.3672 2.1523 2.7656 3.9414zm16.488 109.86c4.3438-0.95312 7.9922-2.6328 10.953-5.0273 2.9531-2.3984 5.0234-5.7188 6.207-9.9648 1.1914-4.2422 1.8945-12.664 1.8945-25.266v-49.344c0-13.297-0.51953-22.207-1.3242-26.734-0.81641-4.5391-2.8438-8.6562-6.082-12.355-3.2461-3.6953-7.9844-6.3516-14.207-7.9648-6.2344-1.6172-16.395-2.4336-34.066-2.4336h-27.23v140.52h44.223c10.191-0.32031 16.738-0.79297 19.633-1.4297z" />
+      <path d="m371.81 163.76c-0.67188 1.7812-3.6172 2.6875-5.8398 2.6875-2.1758 0-3.625-0.86328-4.3594-2.6016-0.73828-1.7266-1.0977-5.6719-1.0977-11.848v-37.137c0-6.3984 0.32031-10.391 0.96875-11.984 0.64062-1.5742 2.0469-2.375 4.2227-2.375 2.2266 0 5.207 0.90234 5.9766 2.7188 0.76172 1.8164 1.1445 5.6992 1.1445 11.641v36.008c-0.23047 7.4023-0.56641 11.695-1.0156 12.891zm-46.465 21.207h32.863c1.3672-5.3672 2.1211-8.3516 2.2734-8.9531 2.9766 3.6016 6.2461 6.3047 9.8398 8.0977 3.5742 1.7969 8.9258 2.6953 13.07 2.6953 5.7695 0 10.746-1.5117 14.945-4.543 4.1914-3.0273 6.8633-6.6094 8-10.73 1.1367-4.1289 1.7031-10.398 1.7031-18.832v-39.422c0-8.4805-0.19141-14.016-0.56641-16.617-0.375-2.6016-1.4961-5.25-3.3594-7.9609-1.8672-2.7109-4.5781-4.8164-8.1289-6.3203-3.5508-1.5039-7.7461-2.2539-12.578-2.2539-4.1992 0-9.5742 0.83984-13.16 2.4961-3.5742 1.6562-6.8203 4.168-9.7344 7.5352v-45.711h-35.168z" />
     </g>
   </Svg>
 );
+
+export const UnrealEngineLogo = () => (
```

Comment:
Can we possibly use the Unreal Engine logo that has the text next to it instead than right below? You can download the `.ai` file [here](https://cdn2.unrealengine.com/BrandingMedia%2FLogos%2FUnreal_Engine_Logos-194aec037564e2e849650161de30d5a294a592ee.zip).

To be a little more specific I'm speaking about this one as it is definitely more visible and pleasant in the format that we are using on our logo scroller

![Unreal_Engine_Horiz_Black](https://user-images.githubusercontent.com/10723208/87365116-b06cfe80-c575-11ea-8ab0-d5cbabdf5537.png)